### PR TITLE
[t1531] fixed scrollbars in timeline/util

### PR DESCRIPTION
**Pull request by secretrobotron**
1. Cleaned up code style a bit
2. Removed timeline/scrollbars
3. Fixed missing scrollbar update in plugin module
4. Rewired timeline to use util/scrollbars

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0ba749bf9ab6e72/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/dae7294003fc5cf/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0ba749bf9ab6e72/output.txt

Total script time: 0.99 mins

Server is running on http://pr623.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/dae7294003fc5cf/output.txt

Total script time: 0.34 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/89e0b2a75052110/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/89e0b2a75052110/output.txt

Total script time: 1.41 mins

Server is running on http://pr623.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/29048649da56523/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b6c223ee57ef626/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/29048649da56523/output.txt

Total script time: 0.75 mins

Server is running on http://pr623.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/b6c223ee57ef626/output.txt

Total script time: 0.55 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/9b65ef60de93430/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/9b65ef60de93430/output.txt

Total script time: 0.54 mins

Server is running on http://pr623.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/80b67fa263ee994/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/80b67fa263ee994/output.txt

Total script time: 0.80 mins

File: src/util/scrollbars.js [hunk 1]
```diff
@@ -183,34 +183,34 @@ define( [ "core/eventmanager" ], function( EventManagerWrapper ){
...
-    } //onMouseDown
+      }
+    }
 
     function setHandlePosition(){
       if( _scrollWidth - _elementWidth > 0 ) {
         _handle.style.left = ( _elementWidth - _handleWidth ) *
-          ( innerElement.scrollLeft / ( _scrollWidth - _elementWidth )) + "px";
-      }else{
+          ( outerElement.scrollLeft / ( _scrollWidth - _elementWidth )) + "px";
```
Comment (mjschranz) [inline src/util/scrollbars.js]: whitespace on the right

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/95d8d6d8c3ce30f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/95d8d6d8c3ce30f/output.txt

Total script time: 0.62 mins

Server is running on http://pr623.butyr.org/templates/test.html

**Pull request merged**